source/nordic_sdk/components/libraries/bootloader_dfu/hci_transport/hci_mem_pool_internal.h

```diff
@@ -42,10 +42,10 @@
 #ifndef MEM_POOL_INTERNAL_H__
 #define MEM_POOL_INTERNAL_H__
 
-#define TX_BUF_SIZE       4u    /**< TX buffer size in bytes. */
-#define RX_BUF_SIZE       32u   /**< RX buffer size in bytes. */
+#define TX_BUF_SIZE       32u    /**< TX buffer size in bytes. */

```

Comment:
Perhaps these values have been used in the bootloader sources.

From: LIYOU ZHOU [mailto:notifications@github.com]
Sent: 15 December 2015 14:51
To: ARMmbed/nrf51-sdk
Cc: Rohit Grover
Subject: Re: [nrf51-sdk] Update files to Nordic sdk 9.0.0 (#8)


In source/nordic_sdk/components/libraries/bootloader_dfu/hci_transport/hci_mem_pool_internal.h<https://github.com/ARMmbed/nrf51-sdk/pull/8#discussion_r47644751>:

> @@ -42,10 +42,10 @@

>  #ifndef MEM_POOL_INTERNAL_H__

>  #define MEM_POOL_INTERNAL_H__

>

> -#define TX_BUF_SIZE       4u    /**< TX buffer size in bytes. */

> -#define RX_BUF_SIZE       32u   /**< RX buffer size in bytes. */

> +#define TX_BUF_SIZE       32u    /**< TX buffer size in bytes. */

@rgrover<https://github.com/rgrover> I can't find how these values are being use. hci_mem_pool_internal.h is being included in components/libraries/hci/hci_mem_pool.c. But I cannot find where components/libraries/hci/hci_mem_pool.h is being used. Can you remember how it was used?

—
Reply to this email directly or view it on GitHub<https://github.com/ARMmbed/nrf51-sdk/pull/8/files#r47644751>.

IMPORTANT NOTICE: The contents of this email and any attachments are confidential and may also be privileged. If you are not the intended recipient, please notify the sender immediately and do not disclose the contents to any other person, use it for any purpose, or store or copy the information in any medium. Thank you.